src/SearchParameters/RefinementList.js

```diff
@@ -74,8 +74,8 @@ var lib = {
    * @param {string} value the value of the refinement
    * @return {RefinementList} a new and updated list
```

Comment:
hah, my "replace all" was a bit too powerful maybe. 

So which function should be deprecated then? I think this is the one you commented on in the built version